test/text/vtt_text_parser_unit.js

```diff
@@ -585,6 +585,19 @@ describe('VttTextParser', () => {
         {periodStart: 0, segmentStart: 0, segmentEnd: 0});
   });
 
+  it('supports extra newlines in cue body', () => {
+    verifyHelper(
+        [
+          {startTime: 20, endTime: 40, payload: ''},
+          {startTime: 40, endTime: 50, payload: 'Test2'},
+        ],
+        'WEBVTT\n\n' +
+        '00:00:20.000 --> 00:00:40.000\n' +
+        '\nTest\n\nExtra line\n\n' +
```

Comment:
Did you really want this text ignored instead of represented in the output?

Reply:
I'd love to display it, but after a lot of noodling I can't see how to do so without using cowboy hijinks counter to other implementations I've seen. For instance, dash.js gives:
`[9751][VTTParser] Skipping cue due to empty/malformed cue text`

The current parser has good adherence to the spec in the implemented areas. I'd like to change that as little as possible while also giving a best effort towards displaying the good parts of a VTT file.